mrbelvedereci/build/handlers.py

```diff
@@ -16,8 +17,9 @@ def queue_build(sender, **kwargs):
         return
 
     # Queue the pending status task
-    res_status = set_github_status.delay(build.id)
-    build.task_id_status_start = res_status.id
+    if settings.GITHUB_STATUS_UPDATES_ENABLED:
+        res_status = set_github_status.delay(build.id) 
+        build.task_id_status_start = res_status.id
```

Comment:
Are `task_id_status_start` and `task_id_status_end` used anywhere? I looked around and didn't see any. Just curious.